tests/auth_tests/test_auth_backends.py

```diff
@@ -194,6 +197,19 @@ def test_authentication_timing(self):
         authenticate(username='no_such_user', password='test')
         self.assertEqual(CountingMD5PasswordHasher.calls, 1)
 
+    def test_authenticate_inactive(self):
+        # Inactive user is rejected by default
+        user = self.UserModel._default_manager.get(pk=self.user.pk)
+        user.is_active = False
+        user.save()
+        self.assertIsNone(authenticate(username='test', password='test'))
+        # User without `is_active` field is allowed
+        with override_settings(AUTH_USER_MODEL='auth_tests.CustomUserWithoutIsActiveField'):
```

Comment:
Ok. Wouldn't it also better to move test to `ModelBackendTest` class and not keep in the base test class, so it won't be executed within every child test class.